src/LogoFX.Client.Core.Platform/src/CommonProperties.cs

```diff
@@ -1,7 +1,13 @@
-﻿#if NET || NETCORE
+﻿
+#if (NET || NETCORE) && !WINUI3
 using System.Windows;
 #endif
 
+using System.Diagnostics.CodeAnalysis;
```

Comment:
Please remove - no need to add dependency on Resharper here

Reply:
It isn't dependency on Resharper.